src/mscorlib/src/System/Security/Permissions/HostProtectionPermission.cs

```diff
@@ -41,8 +41,8 @@ internal enum HostProtectionResource
     [AttributeUsage(AttributeTargets.Method | AttributeTargets.Constructor | AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Assembly | AttributeTargets.Delegate, AllowMultiple = true, Inherited = false )] 
     [System.Runtime.InteropServices.ComVisible(true)]
     [Serializable]
-    // This needs to be in the asmmeta to enable SecAnnotate to successfully resolve and run the security rules. It gets marked
-    // as internal by BCLRewriter so we are simply marking it as FriendAccessAllowed so it stays in the asmmeta.
+    // This needs to be in the asmmeta to enable SecAnnotate to successfully resolve and run the security rules. 
+    // We are simply marking it as FriendAccessAllowed so it stays in the asmmeta.
     [System.Runtime.CompilerServices.FriendAccessAllowedAttribute]
```

Comment:
You can delete this comment as well as FriendAccessAllowedAttribute. None of it is applicable anymore.